src/boot_O2/rand.c

```diff
@@ -1,20 +1,23 @@
 #include "global.h"
 
-// The latest generated random number, used to generate the next number in the sequence.
+//! The latest generated random number, used to generate the next number in the sequence.
 static u32 sRandInt = 1;
 
-// Space to store a value to be re-interpreted as a float.
-// This can't be static because it is used in z_kankyo
+//! Space to store a value to be re-interpreted as a float.
+//! This can't be static because it is used in z_kankyo
 u32 sRandFloat;
```

Comment:
```suggestion
//! This can't be static because it is used in z_kankyo
u32 gRandFloat;
```
If it's used externally, should it have the `g` prefix to indicate that it's global and used across files?